### ISSUE TITLE
"form" attribute for button

### ISSUE THREAD

**Issue by yujonglee** [ArthurClemens/primer_live#203]
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button#attributes

```elixir
# undefined attribute "form" for component PrimerLive.Component.button/1
<Primer.button type="submit" form="form">
  Save
</Primer.button>

<.form id="form">
</.form>
```

**Comment by ArthurClemens** [comment]
The warning indicates that this attribute is not defined as component `attr` or included in `rest`. Functionally it should work though.

**Comment by yujonglee** [comment]
yep - thank you for the fix(9bbcc20)
